src/Core/Framework/Api/ApiDefinition/Generator/Schema/StoreApi/components/parameters/criteria.json

```diff
@@ -0,0 +1,189 @@
+{
+    "components": {
+        "x-parameter-groups": {
```

Comment:
I'm wondering what is the best place to document this custom behaviour 🤔 Is there a way we can put the documentation to this schema definition?

Reply:
what i'm mainly looking for here is docs how to use it

In the most naive way I would add something like "unsage instruction": "Reference parameter groups in your routes params list like this:.."